app/src/main.js

```diff
@@ -6,6 +6,7 @@ import router from './router'
 import store from './store'
 import log from 'loglevel'
 import './registerServiceWorker'
+import vuetify from './plugins/vuetify'
```

Comment:
Use the already imported plugins to bring in the vuetify object. don't do it twice.

Reply:
Done. I also exported vuetify in plugin/index.js as it needs to be bootstrapped on main Vue instance